nerfstudio/model_components/losses.py

```diff
@@ -355,7 +355,8 @@ def forward(
         """
         summed_mask = torch.sum(mask, (1, 2))
         image_loss = torch.sum(self.mse_loss(prediction, target) * mask, (1, 2))
-        image_loss = masked_reduction(image_loss, summed_mask, self.reduction_type)
+        # multiply by 2 magic number?
+        image_loss = masked_reduction(image_loss, 2 * summed_mask, self.reduction_type)
```

Comment:
Why?

Reply:
I have no idea, this was in the original SDFstudio repo and I don't know why it's there. I originally left in to ensure I was correctly implementing the losses/components compared to the original implementation.